src/VSCodeExtension/src/tests/unit/declaration.test.ts

```diff
@@ -1,24 +1,24 @@
 import "mocha";
 import * as assert from "assert";
 import { formatter } from "../../formatter/formatter";
-import { argsRule } from "../../formatter/rules/operation";
+import { argsRule } from "../../formatter/rules/declaration";
 
 describe("arguments rule", () => {
-    it("no error", () => {
+    it("has no error", () => {
         const code = "operation Foo (q : Qubit, n : Int) : Bool";
```

Comment:
This could be done in a separated PR, but did you consider comments?

what if the input if

// operation Foo (q : Qubit, n : Int) : Bool

?

Reply:
Comments are an unhandled case for all rules. Will be addressing this in a future PR.